artemis-commons/src/test/java/org/apache/activemq/artemis/utils/collections/LongHashSetTest.java

```diff
@@ -327,7 +326,7 @@ public void containsEmptySet() {
       final LongHashSet other = new LongHashSet(100);
 
       assertTrue(testSet.containsAll(other));
-      assertTrue(testSet.containsAll((Collection<?>) other));
+      assertTrue(testSet.containsAll(other));
```

Comment:
These aren't redundant casts. The class has its own containsAll(LongHashSet) but really has 2 methods also inherits containsAll(Collection) (unclear if it shouldnt be overriding it).

Notice all the changed lines result in duplicate assertions; this cast is the test trying to force which method is used in order to check both methods.

Reply:
Fixed.